packages/genesys-spark-tokens/snapshot-new/json/gse-flare-dark-semantic.json

```diff
@@ -41,7 +41,7 @@
         }
       },
       "overlay": {
-        "10": "#3e5374"
+        "10": "#3e5374cc"
```

Comment:
This is too long to be a valid hex code I think

Reply:
The extra characters are for the alpha channel.
https://caniuse.com/css-rrggbbaa
https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color